contact/index.html

```diff
@@ -139,44 +142,46 @@ <h2>Get in <br />
-            <p>Join us for an unforgettable hunt in the heart of the wilderness.</p>
-            <p>Explore Africa's beautiful wildlife and discover the magic of the animal kingdom</p>
+            <h3>Royalpaw Safaris.</h3>
+            <img src="../logo.png" alt="Royalpaw Safaris" title="Royalpaw Safaris Logo">
+            <p>Join us for an unforgettable hunt in the heart of the wilderness safaris.</p>
+            <p>Explore Africa's beautiful wildlife and discover the magic of the animal kingdom.</p>
         </div>
     
         <div class="text2">
-            <h3>Get Social</h3>
+            <h3>Get Social.</h3>
             <ul>
-                <li><a href="https://www.facebook.com/royalpawsafarisKE/" target="_blank">Visit us on Facebook</a></li>
-                <li><a href="https://www.facebook.com/royalpawsafarisKE/" target="_blank">View @Twitter Feed</a></li>
-                <li><a href="https://www.tripadvisor.com/UserReviewEdit-g294207-d24113128-Royal_Paw_Safaris-Nairobi.html" target="_blank">Trip Advisor Reviews</a></li>
-                <li><a href="https://instagram.com/royalpawsafariske?igshid=MTJneHFoN2QxdWZlcA==" target="_blank">Follow Our Instagram Page</a></li>
+                <li><a href="https://www.facebook.com/royalpawsafarisKE/" target="_blank">Visit us on Facebook.</a></li>
+                <li><a href="https://www.facebook.com/royalpawsafarisKE/" target="_blank">View our X Feeds.</a></li>
+                <li><a href="https://www.tripadvisor.com/UserReviewEdit-g294207-d24113128-Royal_Paw_Safaris-Nairobi.html" target="_blank">Trip Advisor Reviews.</a></li>
+                <li><a href="https://instagram.com/royalpawsafariske?igshid=MTJneHFoN2QxdWZlcA==" target="_blank">Our Instagram Page.</a></li>
             </ul>
         </div>
         
         <div class="text3">
-            <h3>Contact</h3>
+            <h3>Contact.</h3>
             <ul>
-                <li>Phone Number: +254 745 323 472</li>
-                <li>Phone Number: +254 722 165 462</li>
-                <li>Email Address: info@royalpawsafaris.com</li>
-                <li><a href="../contact/">Reach out form.</a></li>
+                <li>Phone No.: +254 745 323 472</li>
+                <li>Phone No.: +254 722 165 462</li>
+                <li>Email: info@royalpawsafaris.com</li>
+                <li><a href="../contact/">Write to us.</a></li>
             </ul>
         </div>
         
        </footer>
     
        <section class="terms">
+        
             <div class="foot1">
-                <h4><a href="/">Royal Paw Safaris </a></h4>
+                <h4><a href="/">Royalpaw Safaris.</a></h4>
             </div>
-    
             <div class="foot2">
-                <h4>&COPY; 2023 Digital Content Hub | Powered by <a href="mailto:olekingkinga@gmail.com">Olekingkinga Technologies</a></h4>
+                <h4>&COPY; 2024 | Powered by <a href="mailto:olekingkinga@gmail.com">Zion Cloud Technologies Technologies</a></h4>
```

Comment:
Repetition. Remove one word. 
The rest of the content is fine and incredibly well done.